<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MSI" failures="1" tests="1" name="Linux-Firefox" time="13.378" errors="0" timestamp="2025-06-14T01:18:00 IST">
  <testcase classname="testCases.TC_0002_UserCreationReqEmptyField" name="userCreationHappyFlow" time="2.743">
    <failure type="java.lang.AssertionError" message="Test failed due to exception">
      <![CDATA[java.lang.AssertionError: Test failed due to exception
at testCases.TC_0002_UserCreationReqEmptyField.userCreationHappyFlow(TC_0002_UserCreationReqEmptyField.java:106)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- userCreationHappyFlow -->
</testsuite> <!-- Linux-Firefox -->
